### PR TITLE
[JA] Translate querying 1.1.1

### PR DESCRIPTION
**Pull request by starfish719**
I translated `lessons/ecto/querying` section.
Please review! @elixirschool/japanese

refs: https://github.com/elixirschool/elixirschool/commit/950bd0e1b9f0f53bf10cbe0391da88610fd2c691#diff-42858e1c36d5073e0f1f6c9b1b86d979

### CODE REVIEW

File: ja/lessons/ecto/querying.md [hunk 1]
```diff
@@ -90,112 +90,82 @@ iex> Repo.all(query)
...
-    id: 1,
+    id: nil,
     tagline: "Something about video games",
     title: "Ready Player One"
   }
 ]
 ```
 
-`in` ステートメント (`m in Movie`) が _必要ない_ 場合にはクエリ式を使うことができます。データ構造への参照を必要としない場合には `in` ステートメントは必要ありません。上のクエリはデータ構造の参照を必要とません。例えば、特定の条件を満たす映画を選択することはない場合がそれにあたります。そのため `in` 式とキーワードクエリを使う必要はありません。
+返ってくる構造体には `tagline` と `title` フィールドのみが設定されていることに注意してください - これは `select:` の部分の結果です。
```
Comment (riseshia) [inline ja/lessons/ecto/querying.md]:
```suggestion
返ってくる構造体には `tagline` と `title` フィールドのみが設定されていることに注意してください - これは `select:` の結果です。
```

(maybe) I think `部分の` is verbose.

File: ja/lessons/ecto/querying.md [hunk 2]
```diff
@@ -90,112 +90,82 @@ iex> Repo.all(query)
...
 
-### `select` 式の使用
+このようなクエリは、bindingを必要としないほど単純であるため、 *bindingless* と呼ばれます。
 
-クエリのselectステートメントの部分を指定するために `Ecto.Query.select/3` 関数を使います。特定のフィールドだけ選択したい場合は、アトムのリスト、もしくは構造体のキーの参照でそれを指定することができます。1つ目のアプローチを見てみましょう:
+#### クエリ式による `from` の使用
```
Comment (riseshia) [inline ja/lessons/ecto/querying.md]: I want to ask why did you translate `Binding in queries` to this. 🤔

File: ja/lessons/ecto/querying.md [hunk 3]
```diff
@@ -90,112 +90,82 @@ iex> Repo.all(query)
...
+iex> query = from(m in Movie)                                                           
+%Ecto.Query<from m in Example.Movie>
 ```
 
-`from` 関数に渡された第１引数では `in` 式を _使わなかった_ ことがわかります。これは、 `select` とともにキーワードリストを使うためにデータ構造への参照を作る必要が無かったためです。
-
-このアプローチでは、指定されたフィールドである `title` の値のみが格納された構造体が返されます。
-
-2つ目のアプローチでは動きが少しが異なります。この場合は `in` 式を _使う必要があります_ 。これは、映画構造体の `title` キーを指定するためにデータ構造体への参照を作る必要があるためです:
+このような場合には、 `m` を *binding* と呼びます。クエリの他の部分のモジュールを参照できるため、Bindingは非常に便利です。 `id` が `2` より小さい全ての映画のタイトルを選択してみましょう:
```
Comment (riseshia) [inline ja/lessons/ecto/querying.md]:
```suggestion
このような場合には、 `m` を *binding* と呼びます。クエリの他の部分からモジュールが参照できるため、Bindingは非常に便利です。 `id` が `2` より小さい全ての映画のタイトルを選択してみましょう:
```

seems wrong..?

File: ja/lessons/ecto/querying.md [hunk 4]
```diff
@@ -90,112 +90,82 @@ iex> Repo.all(query)
...
+%Ecto.Query<from m in Example.Movie, where: m.id < 2, select: m.title>
 
-15:06:12.752 [debug] QUERY OK source="movies" db=4.5ms queue=0.1ms
+iex> Repo.all(query)                                           
+SELECT m0."title" FROM "movies" AS m0 WHERE (m0."id" < 2) []
 ["Ready Player One"]
 ```
 
-`select` を使うこのアプローチでは、選択した値を含むリストが返されることがわかります。
+ここで非常に重要なことは、クエリの出力がどのように変化したかです。`select:` 部分のbindingで *expression* を使用すると、選択したフィールドが返される方法を正確に指定できます。例えば、タプルを指定できます。
```
Comment (riseshia) [inline ja/lessons/ecto/querying.md]:
```suggestion
ここで非常に重要なことは、クエリの出力がどのように変化したかです。`select:` 部分のbindingで *式* を使用すると、選択したフィールドが返される方法を正確に指定できます。例えば、タプルを指定できます。
```

https://ja.wikipedia.org/wiki/%E5%BC%8F_(%E3%83%97%E3%83%AD%E3%82%B0%E3%83%A9%E3%83%9F%E3%83%B3%E3%82%B0)

File: ja/lessons/ecto/querying.md [hunk 5]
```diff
@@ -90,112 +90,82 @@ iex> Repo.all(query)
...
-    actors: %Ecto.Association.NotLoaded<association :actors is not loaded>,
-    characters: %Ecto.Association.NotLoaded<association :characters is not loaded>,
-    distributor: %Ecto.Association.NotLoaded<association :distributor is not loaded>,
-    id: 1,
-    tagline: "Something about video games",
-    title: "Ready Player One"
-  }
-]
+### マクロクエリ
+上記の例では、 `from` マクロ内でキーワード `select:` と `where:` を使用してクエリを作成しました。これらは、*keyword-basedのクエリ* と呼ばれます。ただし、クエリを作成する別の方法があります。マクロベースのクエリです。Ectoは、`select/3` や `where/3` などにマクロを提供します。各マクロは、*queryable* な値、*明示的なbindingのリスト*、およびアナログなキーワードに提供するのと同じ式を受け入れます:
```
Comment (riseshia) [inline ja/lessons/ecto/querying.md]:
```suggestion
上記の例では、 `from` マクロ内でキーワード `select:` と `where:` を使用してクエリを作成しました。これらは、*keyword-basedのクエリ* と呼ばれます。ただし、クエリを作成する別の方法もあります。マクロベースのクエリです。Ectoは、`select/3` や `where/3` などにマクロを提供します。各マクロは、*queryable* な値、*明示的なbindingのリスト*、およびアナログなキーワードに提供するのと同じ式を受け入れます:
```
(maybe)

Comment (riseshia) [inline ja/lessons/ecto/querying.md]:
```suggestion
上記の例では、 `from` マクロ内でキーワード `select:` と `where:` を使用してクエリを作成しました。これらは、*keyword-basedのクエリ* と呼ばれます。ただし、クエリを作成する別の方法があります。マクロベースのクエリです。Ectoは、`select/3` や `where/3` のようにすべてのキーワードに対してマクロを提供します。各マクロは、*queryable* な値、*明示的なbindingのリスト*、およびアナログなキーワードに提供するのと同じ式を受け入れます:
```

File: ja/lessons/ecto/querying.md (hunk 2 above)
Comment (starfish719) [reply ja/lessons/ecto/querying.md]: @riseshia oops! I mistatke translate.
Maybe,  I looked mixed with other parts.

I fix it 👍 
https://github.com/elixirschool/elixirschool/pull/2166/commits/9eefa9afa68eccf2cefb2e388ac8bfea299b8942

File: ja/lessons/ecto/querying.md (hunk 5 above)
Comment (starfish719) [reply ja/lessons/ecto/querying.md]: https://github.com/elixirschool/elixirschool/pull/2166/commits/7281c610e77caaddcf790aff548d5d27737de743

Comment (starfish719) [reply ja/lessons/ecto/querying.md]: https://github.com/elixirschool/elixirschool/pull/2166/commits/c76b7d0509af177e3fb8023e89cb8c9f9ab9b9e0

**Pull request merged**